tasks/prepare.yml

```
@@ -0,0 +1,17 @@
+---
+
+- name: Add EPEL repository (RedHat)
+  yum:
+    name: "epel-release"
+    state: "present"
+  when: >
+    ansible_os_family == "RedHat" and not
+      (ansible_distribution == 'Amazon' and ansible_distribution_version == '2')
```

Is it possible to break it in a multiline `when`? I think it could be cleaner and more scalable to future conditionals

```suggestion
  when:
    - ansible_os_family == "RedHat"
    - not (ansible_distribution == 'Amazon' and ansible_distribution_version == '2')
```

---

yes! I makes total sense.